packages/zowe-explorer/src/configuration/SettingsConfig.ts

```diff
@@ -84,6 +85,9 @@ export class SettingsConfig {
     }
 
     public static async standardizeSettings(): Promise<void> {
+        // uncomment next line to mock setting being false for dev & test if local storage already been activated.
+        // ZoweLocalStorage.setValue<boolean>(Definitions.LocalStorageKey.SETTINGS_LOCAL_STORAGE_MIGRATED, false);
```

Comment:
Attn all testers!!! 👋🏻 

Uncomment this line for testing the migration steps especially if you have already been migrated to internal storage for v3.